dp/core/ethqueue.c

```diff
@@ -248,3 +248,21 @@ void eth_process_reclaim(void)
 	}
 }
 
+bool eth_rx_idle_wait(uint64_t usecs)
```

Comment:
It is actually already declared here as I didn't change the name
https://github.com/ix-project/ix/blob/master/inc/ix/ethqueue.h#L106

I can move that declaration if it's better